packages/esm-patient-chart-app/src/visit/visit-form/visit-form.scss

```diff
@@ -52,11 +58,8 @@
 }
 
 .tablet {
-  padding: 1rem;
+  padding: 1.5rem 1rem;
```

Comment:
Is this extra padding meant to account for inputs at the bottom of the screen? Such as the `Indication` field in the Medication Order form?